### ISSUE TITLE
mrc prints 'Could not open data file' without saying the file name

### ISSUE THREAD

**Issue by yurivict** [mhekkel/mrc#7]
```
--- CMakeFiles/mkdssp.dir/all ---
Error executing mrc: Could not open data file
--- CMakeFiles/unit-test.dir/all ---
Error executing mrc: Could not open data file
--- CMakeFiles/mkdssp.dir/all ---
*** [mkdssp_rsrc.obj] Error code 1
```

After this it is not clear at all what is the reason of this failure.

**Comment by mhekkel** [comment]
This is not a bug in mrc, but in the CMakeLists.txt file of dssp. Although the error message in mrc could be a bit clearer. Apparently, an empty path was passed as argument. Most likely an empty variable in the cmake file.

Unfortunately, I cannot transfer this issue to a repository in pdb-redo, so I'm closing it here but will have a look there.

**Comment by mhekkel** [comment]
Well, ok, you were right. The error reporting should have been better. Fixed that in release 1.3.6. Along with dropping boost::program_options, finally. Removing yet another cause of headache.
